webrtc.html

```diff
@@ -903,13 +901,87 @@
                     </p>
                   </li>
 
+
+                  
                   <li>
-                    <p>Set <var>connection</var>'s description attribute (
-                    <code><a>localDescription</a></code> or
-                    <code><a>remoteDescription</a></code> depending on the
-                    setting operation) to the
-                    <code><a>RTCSessionDescription</a></code> argument.</p>
-                  </li>
+
+                  <p>Set the <var>connection</var>'s varios description attributes
+                  by executing one of the following. </p>
+                  
+                  <p class="Issue"> NOTE: The principles of pending and current SDP 
+                  were agreed by the WG but the details the this step has not 
+                  yet been fully reviewed. TODO - review this. </p>
+                  
+                  <ul>
+                    
+                    <!-- A transition stable to haveLocalOffer --> <li> <p>If
+                    the local description was set, and the
+                    <code><a>RTCSessionDescription</a></code> argument has a
+                    type of "offer", and it has a version that is later than the
+                    currentLocalDesc, then the pendingLocalDesc will be set to
+                    the argument and the state will transition to
+                    have-local-offer.  </p> </li>
+
+                    <!-- C transition haveRemoteOffer or haveLocalProvAnswer to
+                    stable --> <li><p>If the local description was set, and the
+                    <code><a>RTCSessionDescription</a></code> argument has a
+                    type of "answer", then this complets an offer answer
```

Comment:
The only state this is not expected in is "stable", but if you get it in stable it still might be the best thing to do to just use it vs cause an error. I can go either way on this. Thoughts?